pyproject.toml

```diff
@@ -127,30 +126,30 @@ convention = "google"
 "graders/**/*.py" = [
   # Boolean default positional argument
   "FBT002",
-  "D",
   # Avoid future changes to graders
   "C419",
   "PYI061",
   "RUF036",
   # We are only strict for our own code
```

Comment:
What does "our own code" mean in this context? Aren't graders our own code too?

Reply:
I'll update this comment. I guess I view the graders as "third-party" / interfacing with PL, rather than an integral piece.

Reply:
Changed in [ea72375](https://github.com/PrairieLearn/PrairieLearn/pull/11761/commits/ea723753c7921093977666f0d091782d3d86208f).